src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+/* eslint-disable no-console */
+
+type State = {
+  time: string;
+};
+
+type Props = {
+  clockName: number | string;
+};
+
+export class Clock extends React.Component<Props, State> {
+  state = {
+    time: new Date().toLocaleTimeString(),
+  };
+
+  interval = setInterval(() => {
+    this.setState({ time: new Date().toLocaleTimeString() });
```

Comment:
Using raw Date object in the state will allow you to call `toLocaleTimeString` just once(in the render)